### PR TITLE
Add description to count function

### PR DESCRIPTION
**Pull request by Koziolek**
In scope of #366. Added description of `count/1`, and information about `count/2` function. 

@doomspork

### CODE REVIEW

File: lessons/specifics/ecto.md [hunk 1]
```diff
@@ -187,12 +187,24 @@ In addition to `all/2`, Repo provides a number of callbacks including `one/2`, `
 
 ### Count
 
+If we want to count number users that has confirmed account we could use `count/1`:
```
Comment (nscyclone) [inline lessons/specifics/ecto.md]: It looks like there is a missing preposition: `to count number users` → `to count the number of users`.

Comment (Koziolek) [inline lessons/specifics/ecto.md]: done

**Comment by doomspork** [comment]
Sounds good @Koziolek, thank you! 👍 

We should take note of which translations need to be updated now

**Comment by Koziolek** [comment]
all.

**Pull request merged**